openslides/config/views.py

```diff
@@ -100,3 +101,22 @@ def form_valid(self, form):
             config[key] = form.cleaned_data[key]
         messages.success(self.request, _('%s settings successfully saved.') % _(self.config_collection.title))
         return super(ConfigView, self).form_valid(form)
+
+
+class ConfigViewSet(viewsets.ViewSet):
```

Comment:
At the moment we don't have a policy which config values are public and which not. We can open an issue but for now I have to decide whether to grant read access to all or not. If not nobody could even see the welcome text.